### PR TITLE
Skill Refactor

### PR DESCRIPTION
**Pull request by Jamini**
## About The Pull Request

This PR heavily refactors skill gain, base stats, and 

Some minor bugfixes were also included

## Changelog
```changelog
add: Players can now dynamically set their stat bonuses from education. Up to +15 or -10 in any one stat. COG counts for half.
admin: consolidated our oddity override files.
balance: All oddity stats have been reduced to 30%
balance: All backgrounds have been dramatically reduced in impact. Granting roughly +5 in one or more stats, and -5 in another set of stats.
balance: Ensign MEC 5 > 10
balance: First Officer's now have 15 in all stats
balance: Chaplain now has 35 bio, 20 ROB, 15 TGH, 0 VIG from 25, 0,10,15
balance: Acolyte now has 25 BIO, 10 COG, 10 MEC, 20 ROB, 10 TGH, 0 VIG  from 25,0,20,0,5,10
balance: Argolytes now have 10 COG 15 ROB and 10 TGH from 0,10,15
balance: Custodians now have 10 COG, 10 MEC, 20 ROB, 10 TGH, 0 VIG from 0,0,15,10,15
balance: Club Workers now have a 15 ROB, 15TGH, 10 VIG from 10, 10, 5.
balance: Club Workers now have the Empath perk
balance: Club Managers now hae the Empath and Market Professional Perks
balance: The Chief Engineer now has 40 MEC, 20 COG, 15 TGH, 15 VIG
balance: Ship Engineers now have 35 MEC, 15 COG, 10 TGH
balance: The MErchant now has 15 Rob and 15 TGH, from 10
balance: Union Techs now have 15 Cog from 5
balance: All medical professionals now have 10 VIG
balance: Chemists, Psychiatrists, and Paramedics now have 30 BIO
balance: Scientists now have 20 MEC and 20 COG from 10 and 30
balance:  The RD now has 25 MEC, 40 COG, 25 Bio
balance: Aegis Inspectors now have 25 BIO, 10 COG, and 10 ROB
balance: Medical Specialists now have 5 COG
balance: All oddities now have cash value.
fix: Holsters no longer devour guns
lore: Adjusted captain description
```

### CODE REVIEW

**Comment by Jamini** [comment]
image

File: code/game/jobs/job/captain.dm [hunk 1]
```diff
@@ -23,20 +23,19 @@ var/datum/announcement/minor/captain_announcement = new(do_newscast = 1)
 	ideal_character_age = 70 // Old geezer captains ftw
 	outfit_type = /decl/hierarchy/outfit/job/captain
 
-	description = "You are a NanoTrasen captain. <br>\
+	description = "You are the the Captain. <br>\
```
Comment (Shadow-Quill) [inline code/game/jobs/job/captain.dm:26]: the the

File: code/modules/clothing/accessories/holster.dm [hunk 2]
```diff
@@ -49,7 +49,8 @@
 				SPAN_NOTICE("[user] draws \the [holstered], pointing it at the ground."),
 				SPAN_NOTICE("You draw \the [holstered], pointing it at the ground.")
 				)
-		user.put_in_active_hand(holstered)
+		if(!user.put_in_active_hand(holstered)) //Occulus Edit: If your primary hand is full, draw with your offhand
+			user.put_in_inactive_hand(holstered) //Occulus Edit: Prevents guns from getting deleted with hotkeys.
```
Comment (Shadow-Quill) [inline code/modules/clothing/accessories/holster.dm:53]: `put_in_hands(holstered)`
also bap, atomization >:C
also, just noticed this says "you draw the gun" even if you can't draw the gun (if hands full)

File: zzzz_modular_occulus/code/datums/setup_option/backgrounds/origin.dm [hunk 3]
```diff
@@ -1,120 +1,137 @@
-//As a general rule, all origin backrounds must have summ of +5 of stat values
+//Extreme nerf to backgrounds. No nete bonus
```
Comment (Shadow-Quill) [inline zzzz_modular_occulus/code/datums/setup_option/backgrounds/origin.dm:1]:
```suggestion
//Extreme nerf to backgrounds. No net bonus
```
?

File: zzzz_modular_occulus/code/datums/setup_option/backgrounds/origin.dm [hunk 4]
```diff
@@ -1,120 +1,137 @@
...
+	)
+/*
+/datum/category_item/setup_option/background/origin/medic
+	name = "Medic"
+	desc = "Be it by choice or cosmic design, most of your life has been spent tending to living things: plants, animals, or people. You have a strong understanding of what makes us all tick."
+	stat_modifiers = list(
+		STAT_MEC -5,
+		STAT_BIO = 5
+	)
+*/
```
Comment (Shadow-Quill) [inline zzzz_modular_occulus/code/datums/setup_option/backgrounds/origin.dm:137]: dupe of code immediately above, looks like

**Comment by Jamini** [comment]
image
Including the vote, which passed with a resounding majority.

**Review by Shadow-Quill — CHANGES_REQUESTED** [review]

File: zzzz_modular_occulus/code/datums/setup_option/backgrounds/origin.dm (hunk 4 above)
Comment (Jamini) [reply zzzz_modular_occulus/code/datums/setup_option/backgrounds/origin.dm:137]: It's commented out. Bad indentation

File: code/modules/clothing/accessories/holster.dm (hunk 2 above)
Comment (Jamini) [reply code/modules/clothing/accessories/holster.dm:53]: if(istype(user.get_active_hand(),/obj) && istype(user.get_inactive_hand(),/obj))
		to_chat(user, SPAN_WARNING("You need an empty hand to draw \the [holstered]!"))|

Check above

File: code/modules/client/preference_setup/background/06_training.dm [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+/datum/category_item/player_setup_item/background/education/OnTopic(var/href,var/list/href_list, var/mob/user)
+	if(href_list["biomod"])
+		var/new_bio = input(user, "Enter a value between -10 and 15 for your biology", CHARACTER_PREFERENCE_INPUT_TITLE, pref.BIOMOD) as num|null
+		if(new_bio && CanUseTopic(user))
+			if(calculatetotalpoints() + text2num(new_bio)<16)
+				pref.BIOMOD = max(min(round(text2num(new_bio)), 15), -10)
+			return TOPIC_REFRESH
+
+	else if(href_list["cogmod"])
+		var/new_cog = input(user, "Enter a value between -10 and 15 for your cognition. (COG half the cost of other stats).", CHARACTER_PREFERENCE_INPUT_TITLE, pref.COGMOD) as num|null
```
Comment (Michiyamenotehifunana) [inline code/modules/client/preference_setup/background/06_training.dm:62]: Grammar on this seems a bit weird but meh.

**Review by Michiyamenotehifunana — APPROVED** [review]
Very well atomized PR, 10/10.

Nothing bad stands out to me, should be good.

File: zzzz_modular_occulus/code/datums/setup_option/backgrounds/origin.dm (hunk 4 above)
Comment (Shadow-Quill) [reply zzzz_modular_occulus/code/datums/setup_option/backgrounds/origin.dm]: But why is this here?

**Comment by Jamini** [comment]
> Very well atomized PR, 10/10.
> 
> Nothing bad stands out to me, should be good.

I'll atomize you

**Pull request merged**